neo/Consensus/ConsensusService.cs

```diff
@@ -270,11 +274,30 @@ private void OnConsensusPayload(ConsensusPayload payload)
         private void OnPersistCompleted(Block block)
         {
             Log($"persist block: {block.Hash}");
-            block_received_time = TimeProvider.Current.UtcNow;
+            reference_block_time = TimeProvider.Current.UtcNow;        
+            theoreticalDelay = calcNetworkLatencyOrClockDriftWithMaxOffset(reference_block_time.ToTimestamp(), block.Timestamp);
```

Comment:
Suggest renaming:
* `theoreticalDelay` to `blockReceivedOffset`
* `calcNetworkLatencyOrClockDriftWithMaxOffset` to `CalculateAllowedBlockReceivedOffset`.

Reply:
Both looks good, thanks.